src/main/java/com/codurance/guru/craftspeople/CraftspeopleService.java

```diff
@@ -16,6 +17,6 @@ public Craftsperson retrieveCraftsperson(Integer craftspersonId) {
     }
 
     public List<Craftsperson> retrieveAllCraftsperson() {
-        return repository.findAll();
+        return repository.findAll(Sort.by("firstName"));
```

Comment:
What about the default sorting we agreed on?

Reply:
we should keep `return repository.findAll();` as we agreed on the front end handling sorting, this is missleading